payjoin/src/receive/mod.rs

```diff
@@ -559,27 +577,19 @@ impl ProvisionalProposal {
                 }
             }
         }
-
-        for i in sender_input_indexes {
-            log::trace!("Clearing sender input {}", i);
-            self.payjoin_psbt.inputs[i].non_witness_utxo = None;
-            self.payjoin_psbt.inputs[i].witness_utxo = None;
-            self.payjoin_psbt.inputs[i].final_script_sig = None;
-            self.payjoin_psbt.inputs[i].final_script_witness = None;
-            self.payjoin_psbt.inputs[i].tap_key_sig = None;
-        }
-        Ok(PayjoinProposal {
-            payjoin_psbt: self.payjoin_psbt,
-            owned_vouts: self.owned_vouts,
-            params: self.params,
-        })
+        return sender_input_indexes;
```

Comment:
```suggestion
       sender_input_indexes;
```

Usually we omit the return keyword where possible in rust, at least in this repo